### PR TITLE
Update: toil 5.7.1

### PR DESCRIPTION
**Pull request by mr-c**
Update Toil to v5.7.1, with refreshed dependencies and patch

----
<details>
<summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by mr-c** [comment]
Thanks @priesgo , can you merge this PR?

**Comment by priesgo** [comment]
Looks like the dependencias are well organized in different requirements files, depending on the needs. Are the bioconda dependencies supposed to be the union of all of those or just a minimal subset?

Everything looks good, I'm just curious how you manage that.

**Comment by mr-c** [comment]
> Looks like the dependencias are well organized in different requirements files, depending on the needs. Are the bioconda dependencies supposed to be the union of all of those or just a minimal subset?

My understanding is that conda packages are to include all "extras", so I added all the dependencies for each category that is completely packaged for Conda already.
> 
> Everything looks good, I'm just curious how you manage that.

I'm not the original packager, my personal agenda is that `toil-cwl-runner` is up to date and functional for HPC users in Conda 😀

Advice on how to handle the various extras is welcome. I primarily package for Debian, so I don't keep up to date with everything in (Bio)Conda land

**Comment by priesgo** [comment]
Thanks for your answer. 

But then if you aim to have all these extra dependencies I don't see them all in there. See this for instance https://github.com/DataBiosphere/toil/blob/master/requirements-aws.txt#L3-L7
Or is that on purpose?

That said, I wouldn't know what would be the best approach here, because I understand one may want to have toil but only with some of the extras or with none. Maybe one independent bioconda package per category (eg: toil-base, toil-aws, etc.)? Maybe someone else has prior experience in something similar. Ping @bioconda/core 

Sorry that this is taking long @mr-c

**Comment by mr-c** [comment]
> But then if you aim to have all these extra dependencies I don't see them all in there. See this for instance https://github.com/DataBiosphere/toil/blob/master/requirements-aws.txt#L3-L7
> Or is that on purpose?

That was my mistake; I'll add those extra AWS reqs back in.

> That said, I wouldn't know what would be the best approach here, because I understand one may want to have toil but only with some of the extras or with none. Maybe one independent bioconda package per category (eg: toil-base, toil-aws, etc.)? Maybe someone else has prior experience in something similar. Ping @bioconda/core
> 
> Sorry that this is taking long @mr-c

Can we merge this PR (or a minimal version of it) so that a recent version of Toil is available while the conversation continues?

File: recipes/toil/meta.yaml [hunk 1]
```diff
@@ -15,40 +15,60 @@ build:
...
+    #cwl reqs
+    - cwltool ==3.1.20220628170238
+    - galaxy-tool-util
+    - ruamel.yaml >=0.15,<=0.17.21
+    - ruamel.yaml.clib >=0.2.6
+    - networkx >=2,<2.8.5
+    #htcondor reqs
+    - python-htcondor >=8.6.0,<9
+    #kubernetes reqs
+    - python-kubernetes >=12.0.1.<22
```
Comment (priesgo) [inline recipes/toil/meta.yaml:68]: This line seems to have an extra dot which makes the pipeline fail

**Comment by priesgo** [comment]
> Can we merge this PR (or a minimal version of it) so that a recent version of Toil is available while the conversation continues?

Sure, let's get it to work.

**Review by priesgo — APPROVED** [review]

**Comment by priesgo** [comment]
@BiocondaBot please merge

**Pull request merged**